lib/statsd/instrument/batched_sink.rb

```diff
@@ -153,15 +168,23 @@ def initialize(host, port, buffer_capacity, thread_priority, max_packet_size, st
           @dispatcher_thread = Thread.new { dispatch }
           @pid = Process.pid
           if statistics_interval > 0
-            @statistics = DispatcherStats.new(statistics_interval)
+            type = case @sink.connection
+            when UdsConnection
+              "uds"
+            when UdpConnection
+              "udp"
+            else
+              "unknown"
+            end
```

Comment:
Might as well add a `type` method on `Connection`, the `Sink` shouldn't have to know about the possible implementations.

Reply:
Did that. Thanks!